### PR TITLE
Changes to the Api Key Tab for different views based on role

### PR DESCRIPTION
**Pull request by eoliveros**

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -339,6 +388,9 @@ def is_accessible(self):
         if current_user.has_role('admin'):
             self.can_export = True
             return True
+        if current_user.has_role('merchant'):
```
Comment (djpnewton) [inline models.py:359]: just make it so that all users can export

**Comment by djpnewton** [comment]
I dont see why we need to split ApiKeyViewModel into two different view models, one should be ok

File: models.py (hunk 1 above)
Comment (eoliveros) [reply models.py:359]: remove self.can_export for the role. Added can_export on the function instead so all users can export for merchant tx tab, providing they can access it.

Comment (eoliveros) [reply models.py:359]: removed user and nonce from the api_key

Comment (djpnewton) [reply models.py:359]: just make can_export true for everyone

**Comment by djpnewton** [comment]
can you squash

**Comment by eoliveros** [comment]
closing pull and opening a new one'

**Pull request closed without merging**